template/src/screens/Home.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Should we remove the file `/template/src/screens/.keep` as now we have a `Home.tsx` file in this folder? :)

Reply:
It was removed already 🙇

Reply:
Oh my bad >< Of course I did not read well sorry!